host/4/bookworm/python/python312/python312.Dockerfile

```diff
@@ -73,6 +73,10 @@ COPY --from=python [ "/opt", "/opt" ]
 RUN for file in /opt/python/3.12/bin/*; do \
         ln -sf "$file" /usr/bin/$(basename "$file"); \
     done
+RUN ln -s /opt/python/3.12/lib/libpython3.12.so.1.0 /usr/lib/libpython3.12.so.1.0
+
+# Install opentelemetry packages
+RUN pip install azure-monitor-opentelemetry-exporter azure-monitor-opentelemetry
```

Comment:
Do you need to add any flags like -y for pip? Just wondering if this could cause build to hang waiting for input at some time.